docs/course.md

```diff
@@ -124,6 +124,31 @@ The assessment set order in `infoCourse.json` is the order in which the assessme
     ],
 ```
 
+## Units
+
+Each assessment in the course belongs to a *unit* defined in `infoCourse.json`. Units should be thought of as a unit of instruction, typically with a consistent number of assessments per unit, such as 3 homeworks, 2 labs, and 1 exam. Properties for assessment units are as follows.
```

Comment:
"typically with a consistent number of assessments per unit, such as 3 homeworks, 2 labs, and 1 exam"

I think this sounds a bit restrictive. I'm ok with making this kind of suggestion, but in my courses I have some units that take one class (and thus less homeworks), while other units take a couple of weeks.